Signal/src/ViewControllers/ConversationView/Cells/OWSMessageBubbleView.m

```diff
@@ -0,0 +1,1083 @@
+//
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
+//
+
+#import "OWSMessageBubbleView.h"
+#import "AttachmentUploadView.h"
+#import "ConversationViewItem.h"
+#import "OWSAudioMessageView.h"
+#import "OWSBubbleStrokeView.h"
+#import "OWSBubbleView.h"
+#import "OWSGenericAttachmentView.h"
+#import "OWSMessageTextView.h"
+#import "OWSQuotedMessageView.h"
+#import "Signal-Swift.h"
+#import "UIColor+OWS.h"
+#import <SignalMessaging/UIView+OWS.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface OWSMessageBubbleView ()
+
+@property (nonatomic) OWSBubbleView *bubbleView;
+
+@property (nonatomic) OWSMessageTextView *bodyTextView;
+
+@property (nonatomic, nullable) UIView *lastQuotedMessageView;
```

Comment:
What is the significance of "last" in this name again?

Is there a `firstQuotedMessageView` or any other `quotedMessageViews`?

Same question for `lastBodyMediaView`

Reply:
I've renamed both. Sorry for the confusion.